src/Theme/themeSettings.js

```diff
@@ -54,5 +57,20 @@ const typography = {
 	},
 };
 
-// mui theme settings
-export const themeSettings = { palette, typography };
+export function useTheme() {
+	const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
+
+	return useMemo(
+		() =>
+			responsiveFontSizes(
+				createTheme({
+					palette: {
+						...palette,
+						mode: prefersDarkMode ? 'dark' : 'light',
+					},
+					typography,
+				}),
+			),
```

Comment:
1. The `responsiveFontSizes` wrapper makes your typography responsive!
2. When I spread your theme settings in my previous commit, it was incorrect. `typography` has to be outside `palette`, as seen here

Reply:
Agh, good catch!